lib/sles4sap/azure_cli.pm

```diff
@@ -12,7 +12,7 @@ use testapi;
 use Carp qw(croak);
 use Exporter qw(import);
 use Mojo::JSON qw(decode_json);
-
+use qesapdeployment qw(qesap_az_validate_uuid_pattern);
```

Comment:
it creates a dependency hierarchy that does not sound right to me: what about moving the functio nin the more generic az lib like this

https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20288

Reply:
Yup, agree.